bin/grant-server/main.go

```diff
@@ -176,6 +178,32 @@ func setupRouter(ctx context.Context, logger *zerolog.Logger) (context.Context,
 		r.Mount("/v2/attestations/safetynet", proxyRouter)
 	}
 
+	var roMintPg mint.ReadOnlyDatastore
+	pgMint, err := mint.NewPostgres("", true, "mint_db")
+	if err != nil {
+		raven.CaptureErrorAndWait(err, nil)
+		log.Panic().Err(err).Msg("Must be able to init postgres connection to start")
+	}
+	if len(roDB) > 0 {
+		roMintPg, err = mint.NewPostgres(roDB, false, "mint_read_only_db")
+		if err != nil {
+			raven.CaptureErrorAndWait(err, nil)
+			log.Error().Err(err).Msg("Could not start reader postgres connection")
+		}
+	}
+
+	mintService, err := mint.InitService(pgMint, roMintPg, promotionService)
+	if err != nil {
+		raven.CaptureErrorAndWait(err, nil)
+		log.Panic().Err(err).Msg("unable to init mint service")
+	}
+	if mintService != nil {
+		if os.Getenv("ENV") == "production" {
+			panic("mint cannot exist in production")
```

Comment:
Maybe we could make absolutely sure these endpoints are not active in particular environments through a test.